helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -484,6 +484,10 @@ processors:
     filter:
       node_from_env_var: K8S_NODE_NAME
 
+  {{- include "splunk-otel-collector.k8sAttributesSplunkPlatformMetrics" . | nindent 2 }}
```

Comment:
Would you mind adding this optionally only when Splunk Platform metrics are enabled? I see you are adding the processor to the pipeline conditionally already, setting the same check here would keep the config more streamlined for users not using Splunk platform metrics.

Reply:
Makes sense! Added it